src/components/desktop/desktop-header-full-predictive.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="x-relative x-pb-24">
+    <div class="desktop-width-item">
+      <header class="x-flex x-items-center x-gap-48 x-pt-24">
+        <Logo />
+
+        <div class="x-flex x-flex-1 x-flex-col x-gap-16">
+          <div class="x-relative">
+            <SearchBox />
+          </div>
+        </div>
+
+        <CloseMainModal class="x-button-lead x-button-circle x-button-ghost x-justify-self-end">
+          <CrossIcon class="x-icon-lg" />
+        </CloseMainModal>
+      </header>
+    </div>
+
+    <LocationProvider location="predictive_layer">
+      <FullWidthPredictive />
+    </LocationProvider>
+
+    <div class="desktop-width-item">
+      <div
+        class="x-layout-container x-layout-container-ml-[calc(142px+48px)] x-layout-container-mr-[calc(40px+48px)]"
```

Comment:
I am not sure about placing the related tags with an arbitrary value here, as it can break the alignment if we change the logo or the width of the search input.
Maybe we should create a 3 cols grid to place this items and keep them always aligned.

Reply:
How do you align the logo + searchbar + close icon row and the related tags row when they cant be under the same div? The predictive has to be between them because it has to be over the RL when it opens and it also has to be outside a ny div preventing it from being 100% width.

The RL width also has to be limited for the sliding panel to work, so a regular 3 column setup without fixed values for the first and last columns wouldn't work either.

I couldn't find a better solution for this.